app/events/views.py

```diff
@@ -80,7 +80,17 @@ def attend(request, event_id):
     return redirect("show", event_id=event_id)
 
 
+@login_required
+def leave(request, event_id):
+    if request.method == 'POST':
+        event = get_object_or_404(Event, pk=event_id)
+        user = request.user.profile
+        user.attending_events.remove(event)
```

Comment:
You should also remove the user from the event's attendees, i.e. something like
`event.attendees.remove(user)`